### ISSUE TITLE
Cannot dump from USB Zip drive on Windows

### ISSUE THREAD

**Issue by SilasLaspada** [aaru-dps/Aaru#211]
### Prerequisites

* [X] Are you running the latest version?
* [X] Can you reproduce the problem in the debug version?

### Check and fill as appropiate:
* [X] I was running DiscImageChef under Windows 10...
* [X] ...using .NET Framework 4.7.2...
* [X] ...in 64-bit

### Description

Cannot dump U250 Zip disk from USB Zip drive on Windows 10 with error "Device error 121 trying to guess ideal transfer length." Dumps properly with same hardware on Ubuntu.

### Exact command line used:

.\DiscImageChef.exe dump-media -i G: -o "Test.dicf" -d

### Output of command execution with debug output enabled
```
DiscImageChef 4.5.99.1693-00000000 built by DESKTOP-8B4K7NM in Debug
Copyright c 2011-2018 Natalia Portillo

DEBUG (Dump-Media command): --debug=True
DEBUG (Dump-Media command): --verbose=False
DEBUG (Dump-Media command): --device=G:
DEBUG (Dump-Media command): --stop-on-error=False
DEBUG (Dump-Media command): --force=False
DEBUG (Dump-Media command): --retry-passes=5
DEBUG (Dump-Media command): --persistent=False
DEBUG (Dump-Media command): --resume=True
DEBUG (Dump-Media command): --lead-in=False
DEBUG (Dump-Media command): --encoding=
DEBUG (Dump-Media command): --output=Test.dicf
DEBUG (Dump-Media command): --format=
DEBUG (Dump-Media command): --force=False
DEBUG (Dump-Media command): --options=
DEBUG (Dump-Media command): --cicm-xml=
DEBUG (Dump-Media command): --skip=512
DEBUG (Dump-Media command): --no-metadata=False
DEBUG (Dump-Media command): Parsed options:
DEBUG (SCSI Device): INQUIRY took 2.5139 ms.
DEBUG (ATA Device): IDENTIFY PACKET DEVICE took 3.5039 ms.
Output image format: DiscImageChef format.
DEBUG (SCSI Device): TEST UNIT READY took 0 ms.
DEBUG (SCSI Device): READ CAPACITY took 1668.1049 ms.
DEBUG (SCSI Device): READ (6) took 2316.3732 ms.
DEBUG (SCSI Device): READ (10) took 9.5608 ms.
DEBUG (SCSI Device): READ (12) took 4.5212 ms.
DEBUG (SCSI Device): READ (16) took 5.5348 ms.
DEBUG (SCSI Device): SEEK (6) took 269.8826 ms.
DEBUG (SCSI Device): SEEK (10) took 0 ms.
Using SCSI READ (16) command.
Media has 489532 blocks of 512 bytes/each. (for a total of 250640384 bytes)
DEBUG (SCSI Device): READ (16) took 5.5416 ms.
DEBUG (SCSI Device): READ (16) took 264.1169 ms.
DEBUG (SCSI Device): READ (16) took 270.2215 ms.
DEBUG (SCSI Device): READ (16) took 270.2585 ms.
DEBUG (SCSI Device): READ (16) took 275.5574 ms.
DEBUG (SCSI Device): READ (16) took 269.8451 ms.
Device error 121 trying to guess ideal transfer length.
```

**Comment by SilasLaspada** [comment]
Error still occurs on newest version of DIC, including on a different computer (Windows 7 32-bit). It does dump properly on an Ubuntu computer (tested using dicremote), but displays "Device not ready. Sense UnitAttention ASC 29h ASCQ 00h" before starting to dump, which may be related to the error on Windows.

**Comment by claunia** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Iomega ZIP250USB fails on optimal sector transfer test](https://trello.com/c/KiaQ12Me/17-iomega-zip250usb-fails-on-optimal-sector-transfer-test)

**Comment by SilasLaspada** [comment]
Still present in newest commit.

**Comment by SilasLaspada** [comment]
Fixed as of newest commit.

**Comment by claunia** [comment]
Fixed by 0c0d97e07a429340d00a08a9a13db3757fbcd4d2.
